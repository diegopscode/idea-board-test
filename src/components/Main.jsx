import React from 'react'

const MainApp = ({ children }) => (
	<main className="app-main">
        {children}
	</main>
)

export default MainApp
