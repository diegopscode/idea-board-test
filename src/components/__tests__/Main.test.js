import React from 'react'
import renderer from 'react-test-renderer'

import Main from '../Main'

describe("MainComponent", () => {
    test('snapshot renders main component', () => {
        const component = renderer.create(<Main />)
        let main = component.toJSON()
        expect(main).toMatchSnapshot()
    })
})