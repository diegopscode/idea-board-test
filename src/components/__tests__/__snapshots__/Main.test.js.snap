// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainComponent snapshot renders main component 1`] = `
<main
  className="app-main"
/>
`;
